### PR TITLE
Add price fluctuation, gitignore, makefile, update readme, requirements.txt

### PR DESCRIPTION
**Pull request by maxmohash**
Add price fluctuation, gitignore, makefile, update readme and requirements.txt

### CODE REVIEW

**Comment by maxmohash** [comment]
@shawnshaligram pls review this

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.PHONY: make
+
+
+make:
+		pip install -Ur requirements.txt
```
Comment (shawnshaligram) [inline Makefile]: indentation seems a bit off here

Comment (shawnshaligram) [reply Makefile]: are you sure its `pip install -Ur requirements.txt` and not `pip install -r requirements.txt` ?

Comment (shawnshaligram) [reply Makefile]: i would rename it to `make install_requirements` so its more clear

**Comment by shawnshaligram** [comment]
lgtm @maxmohash other than the comments
 thanks for the update here!!

File: Makefile (hunk 1 above)
Comment (maxmohash) [reply Makefile]: It is for  -U, --upgrade
Upgrade all specified packages to the newest available version. The handling of dependencies depends on the upgrade-strategy used.

https://pip.pypa.io/en/stable/reference/pip_install/#cmdoption-u

I can change the `make` to `make install_requirements` though

Comment (shawnshaligram) [reply Makefile]: fix the indentation on line 5 and then you can rocket

**Pull request merged**